src/utils/validationUtil.ts

```diff
@@ -569,4 +572,30 @@ export class ValidationUtil {
     //     throw new models.HTTPError(400, 'Failed IVA tests must have IVA Defects');
     //   }
   }
+
+  /**
+   * Checks if the given test type ID is present in the CENTRAL_DOCS_TEST_TYPES array
+   * @param testType TestType
+   * @return boolean
+   */
+  public static IsCentralDocTestType({ testTypeId }: TestType): boolean {
+    return CENTRAL_DOCS_TEST_TYPES.includes(testTypeId);
```

Comment:
You should be able to make use of the helper method in commons once the list has been moved over there.